docs/schemas/events/metadatamanager/MetadataStateChange.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "https://raw.githubusercontent.com/umccr/orcabus/main/docs/schemas/events/workflowmanager/WorkflowRunStateChange.schema.json",
+  "description": "EventBridge custom event schema for orcabus.metadatamanager@MetadataStateChange",
+  "title": "AWSEvent",
+  "type": "object",
+  "required": [
+    "detail-type",
+    "detail",
+    "source"
+  ],
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "region": {
+      "type": "string"
+    },
+    "resources": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "source": {
+      "enum": ["orcabus.metadatamanager"]
+    },
+    "time": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "version": {
+      "type": "string"
+    },
+    "account": {
+      "type": "string"
+    },
+    "detail-type": {
+      "enum": ["MetadataStateChange"]
+    },
+    "detail": {
+      "$ref": "#/definitions/MetadataStateChange"
+    }
+  },
+  "definitions": {
+    "MetadataStateChange": {
+      "type": "object",
+      "required": [
+        "model",
+        "action",
+        "data",
+        "refId"
+      ],
+      "properties": {
+        "model": {
+          "type": "string",
+          "enum": [
+            "LIBRARY",
+            "SPECIMEN",
```

Comment:
Until we have a better handle on how specimen work, should this better be `SAMPLE`?

Reply:
Ahh yes, this is probably from the draft before we changed it to `SAMPLE`, I suppose we could just put `LIBRARY` for now as this is the only event we publish.

Reply:
Yup, that works too.